docs/releasenotes/3.0.0.rst

```diff
@@ -1,9 +1,28 @@
-
 3.0.0
-=====
+-----
+
+Deprecations
+============
+
+Several methods that have been marked as deprecated for many releases
+have been removed in this release::
+
+    Image.tostring()
+    Image.fromstring()
+    Image.offset()
+    ImageDraw.setink()
+    ImageDraw.setfill()
+    The ImageFileIO module
+    The ImageFont.FreeTypeFont and ImageFont.truetype ``file`` keyword arg
+    The ImagePalette private _make functions
```

Comment:
Not a change from the previous version, but now I look with again, these shouldn't be in code formatting, because "The" and "module" etc. aren't code.